addon/usables/use-machine.js

```diff
@@ -21,6 +22,7 @@ export class InterpreterService {
 
   setup() {
     this.service = interpret(this.machine, {
+      devTools: DEBUG,
```

Comment:
I'm not aware how `DEBUG` works. Is this true in development and test? Or do users need to set this manually via an env-variable?

Reply:
It's true during dev and test, and false in production.

glimmer/env isn't even a real package, I don't think but something configured by babel during build